src/commons/eventsourcing/kv.rs

```diff
@@ -147,6 +148,11 @@ impl KeyValueStore {
         self.move_key(key, &key.archived())
```

Comment:
This function is no longer called by `store.rs`, I don't see any other references to it now, is it still needed?